### PR TITLE
fix: Unable to save automation "send email to team"

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes https://github.com/chatwoot/chatwoot/issues/6007

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## How Has This Been Tested?
https://www.loom.com/share/fd7948c0cbcb4db89d1066f2e4480e06

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nithindavid — APPROVED** [review]

**Comment by nithindavid** [comment]
@iamsivin Can you please test this on staging and see if emails are getting delivered?

**Review by nithindavid — CHANGES_REQUESTED** [review]
Please fix codeclimate issue

**Comment by iamsivin** [comment]
> Please fix codeclimate issue

Done https://github.com/chatwoot/chatwoot/pull/6052/commits/265203916cb1536825ae596481d571d66e0aa9f2

File: app/javascript/dashboard/helper/actionQueryGenerator.js [hunk 1]
```diff
@@ -23,7 +23,11 @@ const generatePayload = data => {
     if (Array.isArray(item.action_params)) {
       item.action_params = formatArray(item.action_params);
     } else if (typeof item.action_params === 'object') {
-      item.action_params = [item.action_params.id];
+      if (item.action_params.id) {
```
[the remark this answers is not in the document]
Comment (pranavrajs) [reply app/javascript/dashboard/helper/actionQueryGenerator.js]: @fayazara Please test this this out.

**Review by fayazara — APPROVED** [review]
Tested locally, working correctly now 💯

**Review by nithindavid — APPROVED** [review]

**Pull request merged**